ansible/playbooks/roles/upgrade/tasks/opensearch/migrate-odfe-serial.yml

```diff
@@ -0,0 +1,97 @@
+---
+# Below tasks need to be run in serial
+- name: ODFE migr | Stop elasticsearch service
+  systemd:
+    name: elasticsearch
+    enabled: no
```

Comment:
Shouldn't be false?

Reply:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html#parameter-enabled